resources/options_lookup.tsv

```diff
@@ -1036,25 +1036,25 @@ Windows	No Windows	BuildResidentialHPXML	window_ufactor=0.84	window_shgc=0.63	sk
+Water Heater	"Propane Premium, Condensing"	BuildResidentialHPXML	water_heater_type=storage water heater	water_heater_fuel_type=propane	water_heater_location=auto	water_heater_tank_volume=auto	water_heater_heating_capacity=auto	water_heater_efficiency_type=EnergyFactor	water_heater_efficiency_ef=0.82	water_heater_efficiency_uef=0.67	water_heater_recovery_efficiency=0.9	water_heater_standby_loss=0	water_heater_jacket_rvalue=0	water_heater_setpoint_temperature=125				
+Water Heater	Propane Tankless	BuildResidentialHPXML	water_heater_type=instantaneous water heater	water_heater_fuel_type=propane	water_heater_location=auto	water_heater_tank_volume=0	water_heater_heating_capacity=auto	water_heater_efficiency_type=EnergyFactor	water_heater_efficiency_ef=0.82	water_heater_efficiency_uef=0.67	water_heater_recovery_efficiency=0	water_heater_standby_loss=0	water_heater_jacket_rvalue=0	water_heater_setpoint_temperature=125				
+Water Heater	"Electric Heat Pump, 50 gal"	BuildResidentialHPXML	water_heater_type=heat pump water heater	water_heater_fuel_type=electricity	water_heater_location=auto	water_heater_tank_volume=50	water_heater_heating_capacity=auto	water_heater_efficiency_type=EnergyFactor	water_heater_efficiency_ef=2.3	water_heater_efficiency_uef=0.67	water_heater_recovery_efficiency=0	water_heater_standby_loss=0	water_heater_jacket_rvalue=0	water_heater_setpoint_temperature=125				
+Water Heater	"Electric Heat Pump, 50 gal, 140F"	BuildResidentialHPXML	water_heater_type=heat pump water heater	water_heater_fuel_type=electricity	water_heater_location=auto	water_heater_tank_volume=50	water_heater_heating_capacity=auto	water_heater_efficiency_type=EnergyFactor	water_heater_efficiency_ef=2.3	water_heater_efficiency_uef=0.67	water_heater_recovery_efficiency=0	water_heater_standby_loss=0	water_heater_jacket_rvalue=0	water_heater_setpoint_temperature=140				
+Water Heater	"Electric Heat Pump, 80 gal"	BuildResidentialHPXML	water_heater_type=heat pump water heater	water_heater_fuel_type=electricity	water_heater_location=auto	water_heater_tank_volume=80	water_heater_heating_capacity=auto	water_heater_efficiency_type=EnergyFactor	water_heater_efficiency_ef=2.3	water_heater_efficiency_uef=0.67	water_heater_recovery_efficiency=0	water_heater_standby_loss=0	water_heater_jacket_rvalue=0	water_heater_setpoint_temperature=125				
+Water Heater	Other Fuel	BuildResidentialHPXML	water_heater_type=none	water_heater_fuel_type=electricity	water_heater_location=auto	water_heater_tank_volume=0	water_heater_heating_capacity=auto	water_heater_efficiency_type=EnergyFactor	water_heater_efficiency_ef=0.67	water_heater_efficiency_uef=0.67	water_heater_recovery_efficiency=0	water_heater_standby_loss=0	water_heater_jacket_rvalue=0	water_heater_setpoint_temperature=0				

```

Comment:
It's pretty confusing to see UEF values for each water heater that are unused (and don't even make sense given the EF). How about specifying water_heater_efficiency_uef=0 for all of these so it's clear they aren't used? Similar comment applies to the rest of the changes you made in this commit.